src/test/java/seedu/address/model/person/CovidStatusTest.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Test;
+
+public class CovidStatusTest {
```

Comment:
Same as before!

Reply:
I think for test classes, there shouldn't be a need for a description 😄 . I'm following the earlier standards presented in EmailTest and PhoneTest